de.willuhn.Jameica.yaml

```diff
@@ -15,6 +15,8 @@ finish-args:
   - --device=dri
   - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
   - --env=JAVA_HOME=/app/jre
+  - --filesystem=home:rw
+  - --nofilesystem=home/.*
```

Comment:
Could you please remove the `--nofilesystem` line? I'd rather restrict the access again once upstream provides appropriate settings.

Reply:
Done. Sorry for the delay.